source/crud/query-document.txt

```diff
@@ -0,0 +1,197 @@
+===============
+Specify a Query
+===============
+
+.. default-domain:: mongodb
+
+Specify a Query
+~~~~~~~~~~~~~~~
+
+All CRUD operations apply to a set of matched documents in a collection.
+By default, these operations match **all** documents in a collection.
+Sometimes, however, you want to match only certain items in a
+collection; the query document allows you to make your searches more
+specific. Query documents contain one or more field-level expressions
```

Comment:
I like the info in this paragraph! Can we put a little more emphasis upfront on the idea that "you use query expressions to configure which document(s) a query applies to"? I think it could be more effective if we open with that and then use the "by default" case to contrast.

Reply:
I've tried to reword this a couple of times and I can't find a structure I prefer/like as much as the current approach. Unless you can propose a better wording, I think I'd rather keep it the way it is now, but feel free to suggest!